<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1042</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255)</string>
  </property>
  <widget class="QTableWidget" name="table1">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>811</width>
     <height>371</height>
    </rect>
   </property>
   <property name="dragEnabled">
    <bool>true</bool>
   </property>
   <property name="dragDropMode">
    <enum>QAbstractItemView::NoDragDrop</enum>
   </property>
   <property name="defaultDropAction">
    <enum>Qt::CopyAction</enum>
   </property>
   <property name="gridStyle">
    <enum>Qt::SolidLine</enum>
   </property>
   <property name="sortingEnabled">
    <bool>false</bool>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string/>
    </property>
   </column>
   <column>
    <property name="text">
     <string>대학</string>
    </property>
    <property name="background">
     <color>
      <red>242</red>
      <green>255</green>
      <blue>255</blue>
     </color>
    </property>
   </column>
   <column>
    <property name="text">
     <string>개설학과</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>학년</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>이수구분</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>과목명</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>학점-시간</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>담당교수</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>강의시간</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>강의유형</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>비고</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="pushButton_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>91</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color : rgb(238, 238, 238)</string>
   </property>
   <property name="text">
    <string>마법사</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>91</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color : rgb(238, 238, 238)</string>
   </property>
   <property name="text">
    <string>강의 검색</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
